bundle/language_en.properties

```diff
@@ -846,6 +846,7 @@ angal.medicalstock.multiplecharging.preparationdate
 angal.medicalstock.multiplecharging.qtypacket                                                          = Qty/Packet
 angal.medicalstock.multiplecharging.referencenumberabb                                                 = Reference No.
 angal.medicalstock.multiplecharging.selectedlot                                                        = Selected Lot
+angal.medicalstock.multiplecharging.selectedlotwithoutcostpleasespecify                                = Selected Lot {0} without cost, please specify...
```

Comment:
I forgot to save the comment ☹️ and on second thought my alternative wasn't much better (without cost -> has not cost information)

Reply:
I prefer your by the way, more professional, thanks!